roundtripper.go

```diff
@@ -0,0 +1,29 @@
+package aurestclientprometheus
+
+import "net/http"
+
+type PrometheusRoundTripper struct {
+	wrapped http.RoundTripper
+}
+
+func NewPrometheusRoundTripper(wrapped http.RoundTripper) *PrometheusRoundTripper {
+	SetupHttpClientMetrics()
+
+	return &PrometheusRoundTripper{
+		wrapped: wrapped,
+	}
+}
+
+func (vrt *PrometheusRoundTripper) RoundTrip(req *http.Request) (res *http.Response, e error) {
```

Comment:
Named return parameters are never assigned. This means that res and e will always be nil.

Probably safer to remove the response parameter names and just use nil for e in line 18, otherwise "return" without any arguments is valid, but will return nil, nil.